hikari/impl/bot.py

```diff
@@ -23,6 +23,9 @@
 
 from __future__ import annotations
 
+import threading
```

Comment:
The imports seem to have escaped.

The patch I sent earlier wasn't formatted as I didn't have half the stuff installed I needed, so you might want to just double check first.

Reply:
Will do :+1:

Reply:
Looks like they were fine :)